lib/powerpoint.js

```diff
@@ -18,13 +19,19 @@ var debugLogger = debug('pdfppt:debug')
 pdfLogger.log = console.log.bind(console)
 debugLogger.log = console.log.bind(console)
 
+/** allows the caller to provide a path to the Inkscape Executable */
+const INKSCAPE_PATH = process.env.PDF_PPT_INKSCAPE_PATH
+/** allows the caller to provide a path to the MuPdf Executable */
+const MUPDF_PATH = process.env.PDF_PPT_MUPDF_PATH
 /** allows the caller to provide a path to the GhostScript Executable */
 const GS_PATH = process.env.PDF_PPT_GSPATH
 /** allows the caller to provide a path to the ImageMagick Executable */
 const IM_PATH = process.env.PDF_PPT_IMPATH
 
-const gsExecutable = os.platform() === 'win32' ? GS_PATH || 'gswin32c.exe' : 'gs'
-const imageMagickConvert = os.platform() === 'win32' ? IM_PATH || 'convert.exe' : 'convert'
+let gsExecutable = GS_PATH || 'gs'
```

Comment:
Should this be in _setExecutable so all the logic is in one spot?

Reply:
yeah good idea